frontend/src/main/scala/akkaviz/frontend/components/components.scala

```diff
@@ -44,7 +45,27 @@ class ActorSelector(
     val content = div().render
     val stateVar = currentActorState(actor)
     stateVar.trigger {
+      import DOMGlobalScope.Rickshaw
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2145664792/issues/source?bid=3121282&fileBranchId=3151611#l48)